onnx/defs/__init__.py

```diff
@@ -36,12 +44,10 @@ def _Attribute_default_value(self):  # type: ignore
 OpSchema.Attribute.default_value = _Attribute_default_value  # type: ignore
 
 
-def get_functions(domain=ONNX_DOMAIN):  # type: ignore
-    function_map = defaultdict(list)  # type: Dict[int, List[FunctionProto]]
-    function_byte_map = C.get_all_functions(domain)  # type: ignore
-    for function_name, raw_functions in function_byte_map.items():
-        for function_bytes in raw_functions:
-            function_proto = FunctionProto()
-            function_proto.ParseFromString(function_bytes)
-            function_map[function_name].append(function_proto)
-    return function_map
+def get_function_ops():  # type: () -> List[OpSchema]
+    schemas = C.get_all_schemas()
+    function_ops = []
```

Comment:
"return [schema for schema in schemas if schema.has_function_body]" would be more compact.